src/main/java/randoop/main/Minimize.java

```diff
@@ -1028,58 +1041,51 @@ public String call() throws Exception {
    * @param input the {@code String} produced from running a JUnit test suite
    * @return a map from method name to the method's failure stack trace. The stack trace will not
    *     contain any line numbers.
+   * @throws IOException thrown if buffered reader's readLine() fails
    */
-  private static Map<String, String> normalizeJUnitOutput(String input) {
+  private static Map<String, String> normalizeJUnitOutput(String input) throws IOException {
     BufferedReader bufReader = new BufferedReader(new StringReader(input));
```

Comment:
I suggest using try-with-resources so you don't have to call close (and so the reader is closed even if an exception is thrown).